Documentation/MigrationGuide.md

```diff
@@ -1,3 +1,15 @@
+
+# Migration Guides
+
+All the migration guides for SwiftGen are spread out over a few files, depending on the target audience.
+
+| Migration Guide | Target Audience | Abstract |
+|-----------------|-----------------|---------|
+| SwiftGen Migration Guide (this file) | All users of SwiftGen | Changes in configuration, CLI parameters and global changes overview |
+| [Templates Migration Guide](templates/MigrationGuide.md) | Users of SwiftGen's bundled templates (`templateName` option) | Some templates may have been renamed, removed or merged, or their functionality may have changed |
+| [SwiftGenKit Migration Guide](SwiftGenKit Contexts/MigrationGuide.md) | Template writers | Changes in names of variables provided by SwiftGenKit to your templates |
```

Comment:
This link doesn't render (at least when I use the "View" button on GitHub). You probably need to use `%20` there… and everywhere we have a link with that "SwiftGenKit Contexts" folder in it 😉

Reply:
Already fixed 😄

Reply:
Everywhere? I don't see the commit (yet?)

Reply:
I think the GitHub renderer is a bit different than the quicklook plugin I have, because it works locally 🤷‍♂️

Reply:
I amended & force pushed

Reply:
That's why I wanted to avoid spaces in documentation file names whenever possible btw 😛

Reply:
Didn't think you'd be so quick to check 😆

Reply:
Sometimes _I_ can be the Lucky Luke too 😂

Reply:
🔫 🤠